src/ResourceManager/AzureBatch/ChangeLog.md

```diff
@@ -21,7 +21,7 @@
 
 ## Version 4.1.0
 - Added new Get-AzureBatchJobPreparationAndReleaseTaskStatus cmdlet.
-
+- Added byte range start and end to Get-AzureBatchNodeFileContent parameters.
```

Comment:
And fixed the one above too since they are both in the same release I think, it must've been missed in my previous PR